src/definition/messages/IMessageFollowContext.ts

```diff
@@ -0,0 +1,21 @@
+import { IMessage } from '.';
+import { IUser } from '../users';
+
+/**
+ * The context of execution for the following events:
+ * - IPostMessageFollowed
+ */
+export interface IMessageFollowContext {
+    /**
+     * The message that was followed or unfollowed
+     */
+    message: IMessage;
+    /**
+     * The user who follow the message
+     */
+    user: IUser;
+    /**
+     * If the message was followed or unfollowed
+     */
+    isUnfollow: boolean;
```

Comment:
Wouldn't this fall under the same principle as the other PR, no double negatives to make a positive?